src/NuGet.Core/NuGet.Build.Tasks.Pack.Library/Common/MSBuildTaskItem.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
they were public and shared, which caused conflicts. only internal things should be shared.